### ISSUE TITLE
Incorrect Yunohost diagnostic messages post Pi-hole install

### ISSUE THREAD

**Issue by lightnin** [YunoHost-Apps/pihole_ynh#69]
Several hours after installing Pi-hole (latest available, FTLDNS), my yunohost ( 4.2.8.3 (stable) running on rpi4 generated mail quoted below:

> =================================
> Services status check (services)
> =================================
> 
> [ERROR] Service dnsmasq is exited :(
>  - You can try to restart the service, and if it doesn't work, have a look at the service logs in the webadmin (from the command line, you can do this with 'yunohost service restart dnsmasq' and 'yunohost service log dnsmasq').
> 
> [ERROR] Service pihole-FTL is exited :(
>  - You can try to restart the service, and if it doesn't work, have a look at the service logs in the webadmin (from the command line, you can do this with 'yunohost service restart pihole-FTL' and 'yunohost service log pihole-FTL').

Status in the admin panel confirms that Yunohost sees both services disabled. From command line:

> admin@net:~ $ sudo yunohost service log dnsmasq
> journalctl: 
>   - -- Logs begin at Mon 2021-09-20 10:05:21 BST, end at Sat 2021-09-25 19:27:13 BST. --
>   - Sep 25 16:17:40 systemd[1]: Stopping dnsmasq - A lightweight DHCP and caching DNS server...
>   - Sep 25 16:17:41 dnsmasq[1228]: /etc/resolvconf/update.d/libc: Warning: /etc/resolv.conf is not a symbolic link to /etc/resolvconf/run/resolv.conf
>   - Sep 25 16:17:41 dnsmasq[25851]: exiting on receipt of SIGTERM
>   - Sep 25 16:17:41 systemd[1]: dnsmasq.service: Succeeded.
>   - Sep 25 16:17:41 systemd[1]: Stopped dnsmasq - A lightweight DHCP and caching DNS server.
> admin@net:~ $ sudo service dnsmasq status
> ● pihole-FTL.service - LSB: pihole-FTL daemon
>    Loaded: loaded (/etc/init.d/pihole-FTL; generated)
>    Active: active (exited) since Sat 2021-09-25 16:18:15 BST; 3h 9min ago
>      Docs: man:systemd-sysv-generator(8)
>   Process: 3608 ExecStart=/etc/init.d/pihole-FTL start (code=exited, status=0/SUCCESS)

However, the installation seems to still be working - as does Pi-hole, even though pi-hole admin interface reports both dnsmasq and pi-hole services as stopped, right about the time that pi-hole was installed. 

<img width="501" alt="Screenshot 2021-09-25 at 20 42 22" src="https://user-images.githubusercontent.com/1538027/134782874-d38f5828-cd31-4a56-b2a8-6cf1cde83e1c.png">
<img width="330" alt="Screenshot 2021-09-25 at 20 42 33" src="https://user-images.githubusercontent.com/1538027/134782876-fa428998-f753-4a89-b9f7-5401aaa8d55e.png">

Seems like more would be failing if they were really stopped, as my lan is pihole for its DNS? So I'm wondering if this might be an issue with the relationship between the app install process and the internal yunohost automatic diagnosis system?

Thanks for all you are doing to make yunohost freely available. It's important work, and your generosity is very much appreciated.

**Comment by lightnin** [comment]
<img width="643" alt="Screenshot 2021-09-26 at 08 30 28" src="https://user-images.githubusercontent.com/1538027/134796356-bcf815e3-b25e-4585-88aa-992b68b7d651.png">
Got this later in the evening.

**Comment by lightnin** [comment]
And here are logs for pihole.ftl service: https://paste.yunohost.org/mokuricefe

**Comment by lightnin** [comment]
Uninstalled, and dnsmasq no longer exits.

**Comment by lightnin** [comment]
@ericgaspar Do you think the new version you have in testing now might resolve this issue? Assuming my experiences is not unique, Pi-Hole FTLDNS is unusable until this is resolved. Thanks!

**Comment by jhammons** [comment]
@lightnin The testing branch doesn't resolve the issues as far as I can tell. The best I could get working is the FTL serving DNS requests after some tinkering, but additionally the DHCP server functionality is completely broken. I don't think this is related to the packaging, however. I've settled on using the latest version without the DHCP functionality for my own needs, but if I figure out any more I'll report back. Is this package looking for a maintainer?

**Comment by kay0u** [comment]
> Is this package looking for a maintainer?

Yes :-) (But it's not an easy package to maintain because of dnsmasq/FTL)

**Comment by lightnin** [comment]
Thanks for looking into this @jhammons and @kay0u . I'm wondering if this package should be tagged as broken until this is resolved? Alternatively, is there a way to limit functionality in the packaged version to what is compatible with yunohost?

**Comment by munin-m** [comment]
Hello there.
I encounter the same exact problem with this services not working and the DHCP server wich doesn't work.
Was this issues solved?

Thank for your work :)
